client/index.html

```diff
@@ -468,6 +468,53 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
+						<h2>Autocompletion</h2>
+
+						<p>
+							Pressing <kbd>Tab</kbd> will autocomplete nicknames, channels, and <kbd>/</kbd>-commands.
+							Users with previous IRC experience will find this familiar; most clients implement tab-completion.
```

Comment:
I find this a bit unclear. In fact, to select an entry in the autocompletion menu, simply hit <kbd>Tab</kbd> or <kbd>Enter</kbd>, they act the same way when it's open.
The only specific case is the nick one, because it doesn't require a prefix (like # or @), and therefore won't open a dropdown menu when pressing <kbd>Tab</kbd>.

Reply:
How about we just don't mention the old style tab completion? That was only left in for people who have that muscle memory, so it's not something we'd expect people to want to find.

Reply:
That would be fine too, though could it be confusing when autocompletion is disabled yet still available on nicks with <kbd>Tab</kbd>? Maybe we should disable all autocomplete strategies when "Disable autocompletion" is checked...

Reply:
> Note that when autocompletion is disabled, only nicks can be tab-completed.

> Maybe we should disable all autocomplete strategies when "Disable autocompletion" is checked...

Yes, I was meaning to test that because I thought it was the case. For my part, I have disabled the menu-based autocompletion strategy because I find it far too busy/intrusive. The <kbd>Tab</kbd> strategy doesn't have that issue, and I use it often for the items it supports (as mentioned, currently only nicknames).

I don't think the answer is to remove tab-completion when the menu strategy is disabled, though. For missing channel tab-completion, #1433 exists. Should I open another issue to track missing `/`-command tab-complete? Or maybe what's needed is a more general "Unify Auto-Completion Behavior" issue to replace #1433 et al, for which I'd be happy to write an initial proposal to get the discussion started on that front.

Reply:
I don't think it'll be confusing because it's generally not something people will find unless they expect it. I think the way it currently works is good, and I think just ignoring it from the documentation is best.